src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php

```diff
@@ -91,7 +91,6 @@ protected function setUp()
     {
         $this->fs = new Filesystem();
         $this->translationDir = sys_get_temp_dir().'/'.uniqid('sf_translation', true);
-        $this->fs->mkdir($this->translationDir.'/Resources/translations');
```

Comment:
We should have a (legacy) test with this folder created

Reply:
added